.goreleaser.yml

```diff
@@ -73,6 +83,11 @@ nix:
     repository:
       owner: defang-io
       name: defang
+    post_install: |
```

Comment:
Should these be added to the user's shell init too?

Reply:
If I'm not mistaken, the Nix `installShellCompletion` script does that. This section is for Nix only.